src/provider/camunda-platform/properties/IdProps.js

```diff
@@ -0,0 +1,79 @@
+import {
+  getBusinessObject,
+  is
+} from 'bpmn-js/lib/util/ModelUtil';
+
+import { TextFieldEntry, isTextFieldEntryEdited } from '@bpmn-io/properties-panel';
+
+import { useCallback } from '@bpmn-io/properties-panel/preact/hooks';
+
+import {
+  useService
+} from '../../../hooks';
+
+import {
+  isIdValid
+} from '../utils/ValidationUtil';
```

Comment:
Let's reuse the existing util:

```suggestion
import {
  isIdValid
} from '../../bpmn/utils/ValidationUtil';
```

Reply:
The suggested change was incorporated by moving the utils under the shared directory and reused.